plugins/modules/zos_copy.py

```diff
@@ -1977,6 +2032,45 @@ def is_member_wildcard(src):
     )
 
 
+def get_attributes_of_any_dataset_created(
+    dest,
+    src_ds_type,
+    src,
+    src_name,
+    is_binary,
+    volume=None
+):
+    """
+    Get the attributes of dataset created by any case to ensure give the information
+    to the user
+
+    Arguments:
+        dest (str) -- Name of the destination data set.
+        src_ds_type (str) -- Source of the destination data set.
+        src (str) -- Name of the source data set, used as a model when appropiate.
+        src_name (str) -- Extraction of the source name without the member pattern.
+        is_binary (bool) -- Whether the data set will contain binary data.
+        volume (str, optional) -- Volume where the data set should be allocated into.
+
+    Returns:
+        params (dict) -- Parameters created for the dataset created as name, type,
```

Comment:
You're repeating "created" twice, maybe the first time you meant to write "used"?